raster-foundry/raster-foundry
# Standardize S3 client usage

## Overview

This PR is based on https://github.com/raster-foundry/raster-foundry/pull/2565 -- opening this one up to make jenkins less angry. 

This PR standardizes S3 client usage so that the interface now resides in `common`

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

* Go through the process of uploading a GeoTIFF
* Go through the process of supplying a GeoTIFF via S3 URI
* Ensure that both processes complete successfully
* Change organization logo and make sure it still works
* Test other features like exports etc to make sure S3 interactions work as expected


Closes https://github.com/raster-foundry/raster-foundry/issues/1822
